schemas/context/expression.schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/expression",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Expression",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "description": "Any expression which can be used for defining a condition.",
```

Comment:
Since this is the more generic layer I would change the description to talk about an expression for any result type not just a boolean.  Alternatively if you want to restrict it to just boolean expressions I would change the naming to represent that instead of just calling it "expression".

Reply:
You are right, an expression can have multiple types can be very generic. I am removing this data type as it needs more thought. Moving expression fields into segmentdefinition-expression mixin.